src/components/VideoDetailsSidebar/VideoDetailsSidebar.spec.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import VideoDetailsSidebar from './VideoDetailsSidebar.component';
+
+const mockData = {
+  kind: 'youtube#searchListResponse',
+  etag: 'LRviZfd_p3HDDD2uBk5Qv7zaEQU',
+  nextPageToken: 'CBkQAA',
+  regionCode: 'MX',
+  pageInfo: {
+    totalResults: 2323,
+    resultsPerPage: 25,
+  },
+  items: [
+    {
+      kind: 'youtube#searchResult',
+      etag: '_PVKwNJf_qw9nukFeRFOtQ837o0',
+      id: {
+        kind: 'youtube#channel',
+        channelId: 'UCPGzT4wecuWM0BH9mPiulXg',
+      },
+      snippet: {
+        publishedAt: '2014-09-27T01:39:18Z',
+        channelId: 'UCPGzT4wecuWM0BH9mPiulXg',
+        title: 'Wizeline',
+        description:
+          "Wizeline transforms how teams build technology. Its customers accelerate the delivery of innovative products with proven solutions, which combine Wizeline's ...",
+        thumbnails: {
```

Comment:
This is a lot of data and makes hard to know what's is going on in the test, you can move this to it's own file to keep the test clean